client/layout/guided-tours/config-elements.js

```diff
@@ -76,7 +80,7 @@ export class Tour extends Component {
 			return null;
 		}
 
-		return React.cloneElement( nextStep, { isLastStep } );
+		return React.cloneElement( nextStep, { isLastStep, isValid, lastAction } );
```

Comment:
Let's not pass these as props here, since they are made available to all descendants via context.